optuna/logging.py

```diff
@@ -174,7 +174,35 @@ def enable_default_handler() -> None:
 def disable_propagation() -> None:
     """Disable propagation of the library log outputs.
 
-    Note that log propagation is disabled by default.
+    Note that log propagation is disabled by default. You only need to use `disable_propagation`
```

Comment:
`disable_propagation` is rendered in italic and it is not aligned with the other docstrings. How about replace it with 'this method'?
```suggestion
    Note that log propagation is disabled by default. You only need to use this function
```

![image](https://user-images.githubusercontent.com/3255979/113368493-e10df600-9399-11eb-9f28-4c4a329f8514.png)